types/node/v7/index.d.ts

```diff
@@ -1583,9 +1583,12 @@ declare module "readline" {
         completer?: Completer | AsyncCompleter;
         terminal?: boolean;
         historySize?: number;
+        prompt?: string;
+        crlfDelay?: number;
+        removeHistoryDuplicates?: boolean;
     }
 
-    export function createInterface(input: NodeJS.ReadableStream, output?: NodeJS.WritableStream, completer?: Completer | AsyncCompleter, terminal?: boolean): ReadLine;
+    export function createInterface(input: NodeJS.ReadableStream, output?: NodeJS.WritableStream, completer?: Completer | AsyncCompleter, terminal?: boolean, historySize?: number, prompt?: string, crlfDelay?: number, removeHistoryDuplicates?: boolean): ReadLine;
```

Comment:
While the changes to `ReadLineOptions` look correct, there is no overload for `createInterface` that takes more than 4 parameters.